README.md

```diff
@@ -22,7 +22,7 @@ npm install @guidesmiths/license-checker
 - Add a script to run the package
 
 ```
-node_modules/.bin/license-checker --failOn license1,license2
+npx license-checker --failOn license1,license2
```

Comment:
This will call the [original license-checker package](https://www.npmjs.com/package/license-checker) instead [our custom package](https://www.npmjs.com/package/@guidesmiths/license-checker), wouldn't it?

Should be `npx @guidesmiths/license-checker`

Reply:
It's going to call the desired package `@guidesmiths/license-checker`. But if it does not exists from _node_modules/@guidesmiths/license-checker_ it's going to install the original `license-checker` package.
We can avoid it by adding `npx --no-install license-checker` or change the command to `npx @guidesmiths/license-checker`